articles/relnotes/index.md

```diff
@@ -16,6 +16,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: January 28th, 2020*
+
+This release contains the following:
+
+- New Microsoft Quantum .NET Core SDK, see the [README](https://github.com/microsoft/qsharp-compiler/blob/master/src/QuantumSdk/README.md) for more details
```

Comment:
I think this could create some confusion, in that the phrase ".NET Core SDK" has a very specific meaning beyond "an SDK that works with .NET Core." In particular, @bettinaheim's work introduces a new `Microsoft.Quantum.Sdk` NuGet package that can be used to dramatically simplify Q# project files, and that makes it possible to provide neat stuff such as the compiler transformation feature that you mention later.

Reply:
Thanks, I agree, I wasn't sure how to phrase this one. I'll be more specific :)